packages/block-editor/src/components/url-input/style.scss

```diff
@@ -15,11 +15,13 @@ $input-size: 300px;
 			width: $input-size;
 		}
 		padding: $input-padding;
-		border: none;
-		border-radius: 0;
 		margin-left: 0;
 		margin-right: 0;
 
+		&:not(:focus) {
+			border-color: transparent;
```

Comment:
This caused the text link editing popover to have a transparent border on the link combobox:

<img width="355" alt="Link Control with no border on input" src="https://github.com/WordPress/gutenberg/assets/967608/64ed0ee1-a2a6-4fae-9ba9-22bb924e1ba2">

Reply:
Thank you for finding that. Perhaps the increase in CSS selector specificity for adding transparent borders caused this border to disappear unintentionally. In #58505, restoring the default border of the URL input field, i.e. removing this style itself, should resolve this issue.

![image](https://github.com/WordPress/gutenberg/assets/54422211/1146382f-75a0-4f10-95e9-d7412593ec9b)